### ISSUE TITLE
Migrate to 'Key4hep native' algorithms

### ISSUE THREAD

**Issue by BrieucF** [key4hep/CLDConfig#58]
Shall we start migrating the CLD digitization and reconstruction to the newly available Key4hep native solutions? For instance, DDPlanarDigi could already be used from [k4Reco](https://github.com/key4hep/k4Reco/blob/main/k4Reco/DDPlanarDigi/components/DDPlanarDigi.cpp). CLD is a nice candidate to validate these new implementations since it already has working solutions.

**Comment by Zehvogel** [comment]
Good idea. Ideally, we (any volunteers?) should set up more/any validation workflow before :)
